src/routes/users.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const express = require('express');
+const userControler = require('../controllers/users');
```

Comment:
There is a typo in the variable name 'userControler'. It should be 'userController'.